js/server/tests/resilience/shard-distribution-spec.js

```diff
@@ -155,4 +157,164 @@ describe('Shard distribution', function () {
 
   });
 
+  describe("using distributeShardsLike", function () {
+    const followCollection = 'UnitTestDistributionFollower';
+    const numberOfShards = 12;
+
+    const cleanUp = function () {
+      internal.db._drop(followCollection);
+    };
+
+    const compareDistributions = function() {
+      const all = request.get(coordinator.url + '/_admin/cluster/shardDistribution');
+      const dist = JSON.parse(all.body).results;
+      const orig = dist[colName].Current;
+      const fol = dist[followCollection].Current;
+      const origShards = Object.keys(orig).sort();
```

Comment:
I fear this is not correct, we have to make it so that `s99` is before `s101`.

Reply:
Good catch. fixed.